truth.py

```diff
@@ -49,9 +55,21 @@
 
     if ievent%1==0:
         print ("------------------Event", ievent)
+    gluinop4list = []
     for igenpart,genpart in enumerate(genparticles):
         status = genpart.status()
         pdgid = genpart.pdgId()
         if abs(pdgid)!=1000021:
             continue
+        if status !=23:
+            continue
         print(status, pdgid)
+        gluinop4list.append(genpart.p4())
+    gluglu = gluinop4list[0]+gluinop4list[1]
+    h1.Fill(gluglu.Pt())
+    print(gluinop4list[0].E(), gluinop4list[1].E())
+h1.Draw()
+myFile.WriteObject(h1, "MyObject")
```

Comment:
This line could come after the creation of the `TFile` and then the `myFile.MyObject` shouldn't be needed.